src/msg.fbs

```diff
@@ -512,6 +513,7 @@ table Now {}
 
 table NowRes {
   time: uint64;
+  nano: uint32;
```

Comment:
seconds: uint64
subsec_nanos: unit32

Seems reasonable to me. Then convert to millis on the ts-side so `seconds * 1000 + subsec_nanos / 1000`